### PR TITLE
Add metrics tests for zync and zync-que

### PR DESCRIPTION
**Pull request by mdujava**

### CODE REVIEW

File: testsuite/tests/prometheus/zync/test_metrics.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+@pytest.fixture(scope="module")
+def metrics_que(prometheus):
+    """Return all metrics from target defined of que."""
+    metrics = prometheus.get_metrics("que")
+    return metrics
+
+
+@pytest.fixture(scope="module")
+def metrics_zync(prometheus):
```
Comment (pehala) [inline testsuite/tests/prometheus/zync/test_metrics.py:59]: Is there any specific reason why this fixture is not parametrized as well?

Comment (mdujava) [reply testsuite/tests/prometheus/zync/test_metrics.py:59]: There is no reason, will fix it.

File: testsuite/tests/prometheus/zync/test_metrics.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    return _metrics
+
+
+@pytest.mark.parametrize(("pod", "expected_metric"),
+                         [("que", x) for x in METRICS_QUE] + [("zync", x) for x in METRICS_ZYNC]
+                         )
+def test_metric_zync(request, expected_metric, pod):
+    """ Test metrics presense. """
+    actual_metrics = request.getfixturevalue("metrics")(pod)
```
Comment (pehala) [inline testsuite/tests/prometheus/zync/test_metrics.py:66]: Does the fact it calls for the pod for every metric slow it down?

Comment (mdujava) [reply testsuite/tests/prometheus/zync/test_metrics.py:66]: That's right, result is not cached.
It's equivalent to `actual_metrics = prometheus.get_metrics(pod)`.
So is significantly slower then version having separate fixture for each pod, which reuses the prometheus response.

What do you think, should I create 2 fixtures to cache metrics for each pod separately (to speed up tests) 
```
@pytest.fixture(scope="module")
def metrics_que(prometheus):
    return prometheus.get_metrics("que")


@pytest.fixture(scope="module")
def metrics_zync(prometheus):
    return prometheus.get_metrics("zync")
```

Comment (pehala) [reply testsuite/tests/prometheus/zync/test_metrics.py:66]: Yes, that would be the best

**Review by pehala — APPROVED** [review]

**Review by JaurbanRH — APPROVED** [review]

**Pull request merged**